### DISCUSSION TITLE
nationality flag for the entry drivers

### DISCUSSION THREAD

**Discussion by matias918** [assetto-corsa-web/accweb#149] [Ideas]
Hi Marvin, 

first of alll thx for this tool. 
i would like share with you the idea of can set the flag nationality for the entry list.
as example:
{"configVersion":1,"entries":[{"drivers":[{"firstName":"Matias","lastName":"Rivas","shortName":"RIV","driverCategory":0,**_"nationality":14,"_**playerID":"XXXXXXX"}],"raceNumber":918,"forcedCarModel":7,"overrideDriverInfo":1,"isServerAdmin":0,"customCar":"","overrideCarModelForCustomCar":0,"ballastKg":0,"restrictor":0,"defaultGridPosition":-1}],"forceEntryList":1}
Regards

**Comment by Kugelschieber** [comment]
Is this new? It should be hard to add.

**Reply by matias918** [reply]
i  think is added from the beginning, its a extra combo box on the page if you need i could help you making the number with nationality display to make it easier like 14=Argentina .

**Reply by Bubizord** [reply]
> Is this new? It should be hard to add.

It is not. The problem is when you force de Entrylist, the nationality values and the color scheme of the pilot are replaced for the default ones and not what the user specifies (The default one is 12 I think witch is "Other" country). I can add it manually now to the Entrylist, but every change I made from the WebServer it deletes this data because is not a parameter on it.

**Reply by Kugelschieber** [reply]
Sorry, I mean *shouldn't be hard to add.

**Reply by Jovimpc** [reply]
I think it's no new but also think that is not mentioned in the server documentation.

**Comment by Jovimpc** [comment]
I really wish tis could be added, as it was mentioned its a bit frustrating because if you add it manually and then edit something on the entrylist like the car or the ballast, it gets deleted and you have to do it manually all over again, so being like this I prefer to not mess with it.

**Comment by Jovimpc** [comment]
I've searched and found that it only needs to add `"nationality":XX,` in the entrylist after the shortname where XX are the numbers that define the nationality after the "ShortName", the nationalities code list is the following:
```
Other	0
Andorra	49
Argentina	14
Armenia	28
Australia	41
Austria	9
Azerbaijan	50
Bahrain	79
Belgium	7
Brazil	17
Bulgaria	51
Canada	34
Chile	82
China	35
Chinese Taipei	81
Colombia	47
Croatia	33
Cuba	52
Czech Republic	53
Denmark	32
England	86
Estonia	54
Finland	31
France	3
Georgia	55
Germany	2
Great Britain	5
Greece	22
Hong Kong	46
Hungary	6
India	56
Indonesia	38
Iran	78
Ireland	16
Israel	57
Italy	1
Jamaica	58
Japan	48
Kuwait	45
Latvia	59
Lebanon	27
Lithuania	60
Luxembourg	44
Macao	61
Madagascar	84
Malaysia	62
Malta	85
Mexico	29
Monaco	15
Nepal	63
Netherlands	12
New Caledonia	64
New Zeeland	40
Nigeria	65
Northern Ireland	66
Norway	24
Oman	21
Papua New Guinea	67
Philippines	68
Poland	13
Portugal	36
Puerto Rico	19
Qatar	69
Romania	70
Russia	10
San Marino	42
Saudi Arabia	23
Scotland	71
Serbia	72
Singapore	37
Slovakia	20
Slovenia	73
South Africa	18
South Korea	26
Spain	4
Sweden	30
Switzerland	8
Taiwan	74
Thailand	11
Turkey	25
Ukraine	75
United Arab Emirates	43
Uruguay	83
USA	39
Venezuela	76
Wales	77
Zimbabwe	80
```

**Comment by pedrofaria** [comment]
released
